packages/appstore/src/components/wallet-header/__tests__/wallet-header.spec.tsx

```diff
@@ -0,0 +1,315 @@
+            mocked_props.currency = 'EUR';
+            mocked_props.landing_company_name = 'malta';
+            render(
+                <StoreProvider store={mockedRootStore}>
+                    <WalletHeader wallet_account={mocked_props} />
+                </StoreProvider>
+            );
+            const balance_label = screen.getByText('2,345.56 EUR');
+
+            expect(balance_label).toBeInTheDocument();
+        });
+
+        it('Should render balance === 0.00', () => {
+            mocked_props.balance = 0;
+            mocked_props.currency = 'EUR';
+            mocked_props.landing_company_name = 'malta';
+            render(
+                <StoreProvider store={mockedRootStore}>
+                    <WalletHeader wallet_account={mocked_props} />
+                </StoreProvider>
+            );
+            const balance_label = screen.queryByText(`0.00 ${mocked_props.currency}`);
+
+            expect(balance_label).toBeInTheDocument();
+        });
+
+        it('Should render badge Pending verification', () => {
+            getStatusBadgeConfig.mockReturnValue({ icon: '', text: 'Pending verification' });
+
+            mocked_props.balance = 0;
+            mocked_props.currency = 'EUR';
+            mocked_props.landing_company_name = 'malta';
+
+            const mocked_store = mockStore({
+                client: {
+                    loginid: 'MFW1231',
+                },
+                traders_hub: { multipliers_account_status: 'pending', is_eu_user: true },
+            });
+
+            render(
+                <StoreProvider store={mocked_store}>
+                    <WalletHeader wallet_account={mocked_props} />
+                </StoreProvider>
+            );
+
+            const badge = screen.queryByText(/Pending verification/i);
+            const balance_label = screen.queryByText(/balance/i);
+
+            expect(badge).toBeInTheDocument();
+            expect(balance_label).not.toBeInTheDocument();
+        });
+
+        it('Should render badge Verification failed', () => {
+            getStatusBadgeConfig.mockReturnValue({ icon: '', text: 'Verification failed' });
+
+            mocked_props.balance = 0;
+            mocked_props.currency = 'EUR';
+            mocked_props.landing_company_name = 'malta';
+
+            const mocked_store = mockStore({
+                client: {
+                    loginid: 'MFW1231',
+                },
+                traders_hub: { multipliers_account_status: 'failed', is_eu_user: true },
+            });
+
+            render(
+                <StoreProvider store={mocked_store}>
+                    <WalletHeader wallet_account={mocked_props} />
+                </StoreProvider>
+            );
+
+            const badge = screen.queryByText(/Verification failed/i);
+            const balance_label = screen.queryByText(/balance/i);
+
+            expect(badge).toBeInTheDocument();
+            expect(balance_label).not.toBeInTheDocument();
```

Comment:
could we not just use getByText for the badge, and we can just pass the check straight into the expect function:

```suggestion
            expect(screen.getByText(/Verification failed/i);).toBeInTheDocument();
            expect(screen.queryByText(/balance/i);).not.toBeInTheDocument();
```